Chapter08/recipe-06/cxx-example/src/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+add_library(automaton "")
+add_library(evolution "")
+
+include(${CMAKE_CURRENT_LIST_DIR}/conversion/CMakeLists.txt)
```

Comment:
Do you need `${CMAKE_CURRENT_LIST_DIR}` shouldn't the `include` already be relative to the current one?

Reply:
Good point, thanks. This evolved and I forgot to remove it.

Reply:
It seems they are needed. Looks like include is relative to closest main scope.

Reply:
OK, good to know.